src/CatalogPage.js

```diff
@@ -0,0 +1,26 @@
+import React, {Component} from 'react';
+
+import ComponentPage from './ComponentPage';
+import LandingPage from './LandingPage';
+import HeaderBar from './HeaderBar';
+
+import {Switch, Route} from 'react-router';
+
+
+class CatalogPage extends Component {
```

Comment:
Add a comment that this code is NOT in App.js, because TopAppBar and Drawer need separate pages.